app/models/trip.rb

```diff
@@ -9,7 +9,7 @@ class Trip < ApplicationRecord
   has_and_belongs_to_many :students
   has_many :permission_slips
   has_many :required_payments
-
+  attr_accessor :importStudents
```

Comment:
The accessor isn't needed on the model since it will come in through the request params